File: js/prime.js
function prime(num){
    let count = 0;

    for(let i = 2; i < num; i++){
        if(num % i == 0){
            count ++;
            break;
        }
    }

    if(count == 0){
        console.log(num + " is a Prime Number");  
    }else{
        console.log(num + " is Not a Prime Number");
    }
}

prime(6)

File: Problem_Solving/Prime.cpp
#include <iostream>

using namespace std;

int main()
{
    int num, count = 0; 
    cout << "enter number : ";
    cin>> num;

    for(int  i = 2; i<num; i++ ){
        if(num % i == 0 ){
            count = count + 1;
        }
    }
    if(count == 0){
        cout<< num <<" is a Prime number";
    }else{
        cout<< num <<" is not a Prime";
    }
     
}